app/src/main/java/io/horizontalsystems/bankwallet/core/managers/BackgroundRateAlertScheduler.kt

```diff
@@ -19,16 +19,24 @@ class BackgroundRateAlertScheduler(appContext: Context, workerParams: WorkerPara
 
     companion object {
 
+        private const val backgroundRateFetchWorkName = "BackgroundRateFetchWork"
+
         fun startPeriodicWorker(context: Context) {
             val workRequest = createPeriodicWorkRequest()
 
             WorkManager
                     .getInstance(context)
-                    .enqueueUniquePeriodicWork("BackgroundRateFetchWork", ExistingPeriodicWorkPolicy.REPLACE, workRequest)
+                    .enqueueUniquePeriodicWork(backgroundRateFetchWorkName, ExistingPeriodicWorkPolicy.REPLACE, workRequest)
+        }
+
+        fun stopPeriodicWorker(context: Context) {
```

Comment:
May be context (in our case it is always App.instance) should taken directly inside method?